recipes/krb5/bld.bat

```diff
@@ -0,0 +1,15 @@
+:: First, make sure you have sed, gawk, cat, and cp
```

Comment:
@pelson I am not sure how to satisfy those requirements on Windows. `Cygwin`? `MSYS2`? (I stole a patch for `gawk` from anaconda-recipes BTW.)

Ping @gillins who might have an idea on how to proceed here.